app/views/standard/confirmTransferAlreadyConfirmed.scala.html

```diff
@@ -0,0 +1,25 @@
+@import java.util.UUID
```

Comment:
Could you double check with Kirsten and Pia this is the correct design. The JIRA task is a little bit confusing. I think this is correct.